todo_app/data/db_items.py

```diff
@@ -0,0 +1,55 @@
+import pymongo
+import os
+
+class Item:
+    def __init__(self, _id, name, status = 'To Do'):
+        self._id = _id
+        self.name = name
+        self.status = status
+
+    @classmethod
+    def from_db_item(cls, db_item):
+        return cls(db_item['_id'], db_item['name'], db_item['status'])
+    
+def get_primary_connection_string():
+    return os.getenv('PRIMARY_CONNECTION_STRING')
+
+def get_mongo_db_name():
+    return os.getenv('MONGO_DB_NAME')
+    
+def get_mongo_client():
+    connection_string = get_primary_connection_string()
+    client = pymongo.MongoClient(connection_string)
+    return client
+
+def get_todo_items():
+    client = get_mongo_client()
+    db_name = get_mongo_db_name()
+    db = client[db_name]
+    collection = db['todo_items']
```

Comment:
Given that these lines are always (?) going to be combined, would it make sense to have the `get_mongo_client` method actually just give you back the right collection?